src/main/java/org/jenkinsci/plugins/codesonar/CodeSonarPublisher.java

```diff
@@ -367,41 +355,38 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
         
         authenticate(run, baseHubUri, hubInfo.isOpenAPISupported());
 
-        analysisServiceFactory = getAnalysisServiceFactory();
-        analysisServiceFactory.setHubInfo(hubInfo);
-        analysisService = analysisServiceFactory.getAnalysisService(httpService, xmlSerializationService);
-        analysisService.setVisibilityFilter(getVisibilityFilterOrDefault());
-        analysisService.setNewWarningsFilter(getNewWarningsFilterOrDefault());
+        CodeSonarCacheService cacheService = getCodeSonarCacheService(run, hubInfo);
 
-        String analysisId = null;
+        String currentAnalysisId = null;
```

Comment:
If we are going to change this variable name, then maybe we can change it to `currentAnalysisIdString`.  Later there is an 'lCurrentAnalysisId`, which could better be named `currentAnalysisId`, or if you want to be verbose `currentAnalysisIdNumber`.  I think the `lCurrentAnalysisId` naming should be fixed to be more conventional and we should not get in the habit of introducing that sort of notation here.  Not sure where it came from; it was evidently here prior to this PR.